openhtf/conf.py

```diff
@@ -46,6 +47,12 @@
     'This value will override any existing config value at config load time '
     'and will be a string')
 
+ConfigurationDeclaration = (
+    mutablerecords.Record(
+        'ConfigurationDeclaration',
+        ['name'],
+        {'description': None, 'default_value': None, 'optional': True}))
+
 
 class ConfigurationDeclarationError(Exception):
```

Comment:
Does this get raised anywhere else in this file? If not, let's just get ride of the exception type too.  I see you've removed at least one place it was raised (and the associated check went away in the process, not sure that particularly matters, but it is a notable change that you can now have non-optional configuration keys with default values).